src/components/Design/Colors/Primary/PrimaryColor.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Colors from '../../Colors';
+
+const pimaryColor = {
+  'Cyan Dark': '#2B788B',
```

Comment:
They are all keys of the 'primaryColor' Object, you should use maybe camel case naming convention for the keys. Except this, the type of keys are already String type, you should remove all single quotes here.
For example: 
cyanDark: '#BBBBBB',

Reply:
Hi Young,

"PrimaryColor" keys are updated. Thanks.